composer.json

```diff
@@ -23,11 +23,11 @@
     "require": {
         "php": ">=7.2",
         "kkszymanowski/traitor": "^0.2.0",
-        "laravel/framework": "~6.0|~7.0|~8.0"
+        "laravel/framework": "~6.0|~7.0|~8.0|^9.0"
```

Comment:
wouldn't be better to use caret ^ for all versions? more consistent & cleaner i suppose.